src/main/scala/org/constellation/consensus/RoundManager.scala

```diff
@@ -37,6 +37,7 @@ class RoundManager(roundTimeout: FiniteDuration)(implicit dao: DAO)
     case StartNewBlockCreationRound if !ownRoundInProgress =>
       ownRoundInProgress = true
       createRoundData(dao).fold {
+        println("------------------ ERROR create round data")
```

Comment:
you want this to stay in? Any reason not to use logger?

Reply:
it was for debugging only, cleaned up right now.